lib/src/is_mounted.dart

```diff
@@ -0,0 +1,38 @@
+part of 'hooks.dart';
+
+/// Returns an [IsMounted] object that you can use
+/// to check if the [State] is mounted.
+/// ```dart
+/// final isMounted = useIsMounted();
+/// useEffect((){
+///   myFuture.then((){
+///     if (isMounted()) {
+///       // Do something
+///     }
+///   });
+///   return null;
+/// }, []);
+/// ```
+/// See also:
+///   * The [State.mounted] property.
+IsMounted useIsMounted() {
```

Comment:
Could you use the class syntax instead?
Internal hooks all use classes for better hot-reload

Reply:
Oh, I didn't know that. Sure, I'll change it!